doc/lua_api.txt

```diff
@@ -5356,6 +5358,21 @@ Can be obtained via `minetest.get_meta(pos)`.
   meaning it's best to call this when initializing all other meta (e.g.
   `on_construct`).
 
+`BlockMetaRef`
+-------------
+
+Node metadata: reference extra data and functionality stored in a mapblock.
+Can be obtained via `minetest.get_block_meta(blockpos)`.
+
+### Methods
+
+* All methods in MetaDataRef
+* `mark_as_private(name or {name1, name2, ...})`: Mark specific vars as private
```

Comment:
Unlike in node meta, there's currently no way the client could make use of this functionality.
Metadata fields can be inserted in formspecs using `${METAFIELD}`, but there's no such thing for mapblock metadata.

FYI.

Reply:
Should I make block meta fields private by default? (Related: #8687)

Reply:
Yes, and at the moment there's zero use of this function.

Reply:
But it might get useful in the future, for sscsm.

Reply:
Done.